ios/Sources/Component/View/FeedContentListView.swift

```diff
@@ -38,7 +38,6 @@ public struct FeedContentListView: View {
             }
         }
         .padding(.horizontal, SmallCard.Const.margin)
-        .animation(.easeInOut)
```

Comment:
First time show mediaScreen, it has rattling with search bar and feedlist.
Do you know anything about this?

Reply:
I notice that, but it seems like an issue of iOS when combining SwiftUI with UIKit.
This line can improve the rattling but also introduce other issues:
- Unnecessary animation when scrolling to make the navigation bar become the inline style
- Break the navigation appearance when setting the background color by SwiftUI (may be solved by embedding them in an extra `ZStack`)

So I think may be better to ignore the rattling.